src/analysis/normed_space/bounded_linear_maps.lean

```diff
@@ -228,6 +228,25 @@ end
 
 section bilinear_map
 
+namespace continuous_linear_map
+
+lemma map_add₂ (f : E →L[𝕜] F →L[𝕜] G) (x x' : E) (y : F) : f (x + x') y = f x y + f x' y :=
+by rw [f.map_add, add_apply]
+
+lemma map_sub₂ (f : E →L[𝕜] F →L[𝕜] G) (x x' : E) (y : F) : f (x - x') y = f x y - f x' y :=
+by rw [f.map_sub, sub_apply]
+
+lemma map_neg₂ (f : E →L[𝕜] F →L[𝕜] G) (x : E) (y : F) : f (- x) y = - f x y :=
+by rw [f.map_neg, neg_apply]
+
+lemma map_smul₂ (f : E →L[𝕜] F →L[𝕜] G) (c : 𝕜) (x : E) (y : F) : f (c • x) y = c • f x y :=
+by rw [f.map_smul, smul_apply]
+
+lemma map_zero₂ (f : E →L[𝕜] F →L[𝕜] G) (y : F) : f 0 y = 0 :=
+by rw [f.map_zero, zero_apply]
+
+end continuous_linear_map
+
```

Comment:
Can you generalize these to the semilinear case (as in the corresponding `linear_map` versions of these lemmas in `linear_algebra/bilinear_map.lean`)? Also, these lemmas shouldn't require a norm, just a topological space should be enough.

Reply:
I did it, it is good to generalize to the semilinear map case (thanks for the reference to the `bilinear_map`).
To get a topology on `F →L[𝕜] G` we use [`continuous_linear_map.to_normed_group`](https://leanprover-community.github.io/mathlib_docs/analysis/normed_space/operator_norm.html#continuous_linear_map.to_normed_group), so a norm is necessary, at least for `F` and `G`. I weakened the type-class arguments a little bit, but honestly that is a little silly because of the current asymmetry between `E` and `F`.

Reply:
Ah yes -- sorry for the confusion!